### ISSUE TITLE
Wrong SVG shape being drawn

### ISSUE THREAD

**Issue by samzmann** [LaVielle/contoursWeb#2]
For some reason that remains obscure to me, contours.js is not drawing the right shape around the blobs.
When I open the SVG in Illustrator, I see that strokes are being drawn around each group of 4 pixels (1px heigh 4px width) is some areas, and that the blobs are detected only in some regions.
Because of this, and because the minimum stroke weight is 1pt, it appears like blobs are being filled a certain color, and not blobs filled in another colors, however it is the strokes that are fat and give the illusion of a fill.
See the attached zip file and open it in Illustrator for a visual explanation.
[svgTest3.svg.zip](https://github.com/LaVielle/contoursWeb/files/751027/svgTest3.svg.zip)

To sum up:

- Why are strokes being drawn around 4x1 groups of pixels?
- Why are only some blobs being detected by contours.js?
